packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -374,8 +376,22 @@ export class TeamsAppSolution implements Solution {
           await fs.copy(readme, `${ctx.root}/README.md`);
         }
       }
-    } else {
-      return res;
+
+      const azureResources = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
+        ?.azureResources;
+      const hasBackend = azureResources?.includes(AzureResourceFunction.id);
```

Comment:
do we need to check if `azureResources` is null here?

Reply:
Yes we do. Already checked.